vignettes/tidy-data.Rmd

```diff
@@ -54,9 +54,10 @@ A tidy version of the pregnancy data looks like this: (you'll learn how the func
 ```{r, message = FALSE}
 library(tidyr)
 library(dplyr)
-preg2 <- preg %>% 
-  gather(treatment, n, treatmenta:treatmentb) %>%
-  mutate(treatment = gsub("treatment", "", treatment)) %>%
+library(stringr)
+preg2 <-preg %>% 
+  pivot_longer(cols = c(treatmenta:treatmentb), names_to = "treatment", values_to = "n") %>% 
```

Comment:
Can you please omit the name of the `cols` argument?

Reply:
I assume that you mean `pivot_longer((treatmenta:treatmentb), names_to = "treatment", values_to = "n")`, if so, then done.